src/main/resources/assets/integrateddynamics/lang/en_US.lang

```diff
@@ -807,6 +809,95 @@ operator.operators.integrateddynamics.cast.name=Cast
+config.integrateddynamics.biome=Biome generation settings
+config.integrateddynamics.biome.tooltip=Change biome generation settings
+config.integrateddynamics.machine=Machine settings
+config.integrateddynamics.machine.tooltip=Change machine settings
+config.integrateddynamics.potion=Potion settings
+config.integrateddynamics.potion.tooltip=Change potion settings
+config.integrateddynamics.core=Core settings
+config.integrateddynamics.core.tooltip=Change core settings
+config.integrateddynamics.modcompat=Mod compatibility settings
+config.integrateddynamics.modcompat.tooltip=Change mod compatibility settings
+config.integrateddynamics.capability=Capability settings
+config.integrateddynamics.capability.tooltip=Change capability settings
+
+# Configuration
+config.integrateddynamics=%s configuration
+
+## Biome settings
+config.integrateddynamics.biomeMeneglin=Enable Meneglin biome
+config.integrateddynamics.biomeMeneglin.spawnWeight=Meneglin biome Spawn Weight
+
+## Block settings
+config.integrateddynamics.menrilLeaves.berriesDropChance=Berries drop chance
+
+## Capability settings
+capability.integrateddynamics.cableFakeable=Allow fake cables
+capability.integrateddynamics.dynamicLight=Enable dynamic lighting
+capability.integrateddynamics.dynamicRedstone=Enable dynamic redstone
+capability.integrateddynamics.energyNetwork=Enable energy network
+capability.integrateddynamics.facadeable=Allow Facades
+capability.integrateddynamics.networkCarrier=Enable network carrier
+capability.integrateddynamics.networkElementProvider=Enable network element provider
+capability.integrateddynamics.partContainer=Enable part containers
+capability.integrateddynamics.partNetwork=Enable part network
+capability.integrateddynamics.pathElementProvider=Enable path element provider
+capability.integrateddynamics.variableContainer=Enable variable container
+capability.integrateddynamics.variableFacadeHolder=Allow variable facade holder
+
+## Core settings
+config.integrateddynamics.general.analytics=Send anonymous analytics
+config.integrateddynamics.general.cacheCableModels=Cache cable models
+config.integrateddynamics.general.versionChecker=Enable version check
+
+## General settings
+config.integrateddynamics.general.energyConsumptionMultiplier=Energy consumption multiplier
+config.integrateddynamics.general.partOverlayRenderdistance=Part overlay render distance
+
+## Item settings
+config.integrateddynamics.variable=Enable Variable Card
+config.integrateddynamics.variableTransformer=Allow Variable Transformer
+config.integrateddynamics.menrilBerries.nightVision=Enable Menril Berry nightvision
+
+## Machine settings
+config.integrateddynamics.energyBattery.capacity=Battery Capacity
+config.integrateddynamics.energyBattery.energyPerTick=Energy per Tick
+config.integrateddynamics.general.defaultPartUpdateFreq=Part update frequency
+
+## Mod compatibiliy settings
+config.integrateddynamics.CharsetPipes=Charset Pipes aspects
```

Comment:
I would remove the "aspects" here. Who knows, in the future we may also add some operators.